src/GRPCCodes.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+namespace Vendasta\Vax;
+
+/**
+ * Class GRPCCodes If this is wrong, PHP does not match GO
```

Comment:
Shamelessly stolen from GO since I couldn't find this information ANYWHERE for php.